### ISSUE TITLE
SSL doesn't seem to work

### ISSUE THREAD

**Issue by teleharry** [milux/ctldap#41]
The options seems to be missing in `const server = ldap.createServer();`
So, ldaps doesn't work, even ldapCertFilename and ldapKeyFilename ist defined.

Could you change it for future release?

```
if (config.ldapCertFilename && config.ldapKeyFilename) {
  const ldapCert = fs.readFileSync(new URL(`./${config.ldapCertFilename}`, import.meta.url), { encoding: "utf8" }),
      ldapKey = fs.readFileSync(new URL(`./${config.ldapKeyFilename}`, import.meta.url), { encoding: "utf8" });
  options = { certificate: ldapCert, key: ldapKey };
}
const server = ldap.createServer(options);
```

**Comment by milux** [comment]
If you already have fixed that, could you provide us a PR please? :)

**Comment by milux** [comment]
Fixed in version 3.1.0.
